pkg/test/framework/components/istio/operator.go

```diff
@@ -202,27 +204,31 @@ func deployControlPlane(c *operatorComponent, cfg Config, cluster kube.Cluster,
 	if err != nil {
 		return err
 	}
-	cmd := []string{
-		"manifest", "apply",
-		"--skip-confirmation",
-		"--logtostderr",
+
+	c.installSettings = []string{
 		"-f", iopFile,
 		"--set", "values.global.imagePullPolicy=" + s.PullPolicy,
-		"--wait",
 	}
 	// If control plane values set, assume this includes the full set of values, and .Values is
 	// just for helm use case. Otherwise, include all values.
 	if cfg.ControlPlaneValues == "" {
 		for k, v := range cfg.Values {
-			cmd = append(cmd, "--set", fmt.Sprintf("values.%s=%s", k, v))
+			c.installSettings = append(c.installSettings, "--set", fmt.Sprintf("values.%s=%s", k, v))
 		}
 	}
-
 	if c.environment.IsMulticluster() {
 		// Set the clusterName for the local cluster.
 		// This MUST match the clusterName in the remote secret for this cluster.
-		cmd = append(cmd, "--set", "values.global.multiCluster.clusterName="+cluster.Name())
+		c.installSettings = append(c.installSettings, "--set", "values.global.multiCluster.clusterName="+cluster.Name())
+	}
+
+	cmd := []string{
```

Comment:
Do you want to run generate here first and save it off for the `Close`? I assume you can easily apply a generated config?

Reply:
Good idea, I changed it to do this, and fixed how MC behaves, I think the way it was would behave weird with MC since we call append multiple times